aibolit/scripts/readability.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import multiprocessing
+import os
+import subprocess
+import time
+from multiprocessing import Pool
+from pathlib import Path
+import csv
+
+# You need to download the archive here:
+# https://dibt.unimol.it/report/readability/files/readability.zip
+# Unzip and put it into the executable's current directory
+
+parser = argparse.ArgumentParser(description='Compute Readability score')
+parser.add_argument('--filename',
+                    help='path for file with a list of Java files',
+                    required=True)
+
+args = parser.parse_args()
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+results = {}
+
+path = '05'
+os.makedirs(path, exist_ok=True)
+csv_file = open('05/05_readability.csv', 'w', newline='\n')
```

Comment:
@lyriccoder why are we opening it here? Let's open it in `log_result` using `with` statement:

https://docs.python.org/2.5/whatsnew/pep-343.html

Reply:
BECAUSE WE SHOULDN'T CLOSE FILE EVERY TIME. https://docs.python.org/3/tutorial/inputoutput.html. 

`with open` closes file automatically. I just add some lines and need to flush to a file not to loose results.

Reply:
@lyriccoder ok, where did you close the file? :)